.github/workflows/actions.yml

```diff
@@ -43,9 +101,15 @@ jobs:
     - name: Install tox
       run: |
         python -m pip install --upgrade tox
+    - name: Get Tox Environment Name From Matrix Name
+      uses: rishabhgupta/split-by@v1
+      id: split
```

Comment:
nitpick: can we give this a more verbose name? Something like `split-matrix-name`

Reply:
sure